### PR TITLE
Steps to install webpack

### PR DESCRIPTION
**Pull request by Kratosmsn11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -284,3 +284,13 @@ Finally, we run and wait for review.
 ```sh
 git pull
 ```
+
+
+## TLDR
+
+- add webpack
+- create a webpack.config.js file
+- edit the webpack config file
```
Comment (miorel) [inline README.md]:
```suggestion
- edit the webpack config file to match how your project is organized
```

File: README.md [hunk 2]
```diff
@@ -284,3 +284,13 @@ Finally, we run and wait for review.
...
+- set the location of the packages in webpack config file
```
Comment (miorel) [inline README.md]: Is this line different from the previous line? If not, remove?

**Review by miorel — APPROVED** [review]
Thank you for writing this up! Let's get it in and add we can add details later.

File: README.md (hunk 2 above)
Comment (Kratosmsn11) [reply README.md]: this is just a step mentioned as per my reference, as what all we did in order. Will make the necessary changes in the final draft.

**Pull request merged**